tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecWorkflowTool/SpecWorkFlowTool.cs

```diff
@@ -389,8 +389,19 @@ public async Task<string> GetSDKPullRequestDetails(string language, int workItem
                     return $"SDK generation pipeline did not succeed. Status: {pipeline.Result?.ToString()}. For more details: {DevOpsService.GetPipelineUrl(buildId)}";
                 }
 
-                var data = await devopsService.GetSDKPullRequestFromPipelineRunAsync(buildId, language, workItemId);
-                return data;
+                var pr = await devopsService.GetSDKPullRequestFromPipelineRunAsync(buildId, language, workItemId);
+
+                if (gitHelper.IsPullRequestUrl(pr))
```

Comment:
You can avoid this check by using one regex to parse out all 3 components instead of having a helper in git. If regex parser fails, skip updating the info.